src/components/Map/markerStyle.tsx

```diff
@@ -7,36 +7,44 @@ import {ColorScheme} from '../Util/Colors';
  * @param hover - boolean: is the cursor hovering over this marker?
  * @param selected - boolean: is this cursor currently selected?
  * @param colors - the current color scheme being used
+ * @param isValid - boolean: does the sensor have a valid AQI
  */
 export function createSensorIcon(
   aqiReading: string,
   hover: boolean,
   selected: boolean,
-  colors: ColorScheme
+  colors: ColorScheme,
+  isValid: boolean
 ): H.map.Icon {
   let color = colors.good; // Initialize color
-  const aqi = Number(aqiReading);
-  if (aqi < aqiCutoffs.good) {
-    color = colors.good;
-  } else if (aqi < aqiCutoffs.moderate) {
-    color = colors.moderate;
-  } else if (aqi < aqiCutoffs.sensitive) {
-    color = colors.sensitive;
-  } else if (aqi < aqiCutoffs.unhealthy) {
-    color = colors.unhealthy;
-  } else if (aqi < aqiCutoffs.veryUnhealthy) {
-    color = colors.veryUnhealthy;
+  if (isValid) {
+    const aqi = Number(aqiReading);
+    if (aqi < aqiCutoffs.good) {
+      color = colors.good;
+    } else if (aqi < aqiCutoffs.moderate) {
+      color = colors.moderate;
+    } else if (aqi < aqiCutoffs.sensitive) {
+      color = colors.sensitive;
+    } else if (aqi < aqiCutoffs.unhealthy) {
+      color = colors.unhealthy;
+    } else if (aqi < aqiCutoffs.veryUnhealthy) {
+      color = colors.veryUnhealthy;
+    } else {
+      color = colors.hazardous;
+    }
   } else {
-    color = colors.hazardous;
+    color = {backgroundColor: '#888888', textColor: 'black'};
```

Comment:
Maybe put a `const InactiveSensor: color = {backgroundColor: '#888888', textColor: 'black'};` in the `Colors.ts` (just like a singular exported static color) and use that?